### PR TITLE
Add Tests For cycleway:BACKWARD=lane

### PR DESCRIPTION
**Pull request by droogmic**

### CODE REVIEW

**Comment by droogmic** [comment]
For https://github.com/a-b-street/osm2lanes/issues/161

File: data/tests.yml [hunk 1]
```diff
@@ -730,6 +730,85 @@
...
+    lanes:
+      - type: travel
+        designated: foot
+      # - type: separator
+      #   markings:
+      #     - style: kerb
+      - type: travel
+        direction: backward
+        designated: bicycle
+      - type: separator
```
Comment (dabreegster) [inline data/tests.yml:759]: Wait, why're some of the separators enabled and some commented out?

**Review by dabreegster — APPROVED** [review]
I got lost following the discussion in the issue a while ago... but tests locking in our declaration of correctness are good

File: data/tests.yml (hunk 1 above)
Comment (droogmic) [reply data/tests.yml:759]: that one is clear, the schema for kerbs is still up in the air...

reminds me that I need to fix that :P

**Pull request merged**